src/test/resources/pages/instructorFeedbackResultsEditCommentByDifferentInstructor.html

```diff
@@ -2169,7 +2169,7 @@ <h4 class="modal-title">
                                     <span class="text-muted">
                                       From: Teammates Test [${datetime.now} SGT] (last edited by Teammates Test at ${datetime.now} SGT)
                                     </span>
-                                    <span class="glyphicon glyphicon-eye-open" data-original-title="This response comment is visible to response giver, instructors, response recipient, other students in this course, an" data-placement="top" data-toggle="tooltip" style="margin-left: 5px;" title="">
```

Comment:
It happened before but probably it went unnoticed.
Reason for this sort of tooltip is insufficient viewers covered in the method. (I can fix this in another PR if you say so)

And the ones you pointed out, changing of `response giver` to `nobody` was because of `isVisibilityFollowingFeedbackQuestion` (This would take care of current changes you pointed out to in this PR)